### ISSUE TITLE
Virtualenv generator - Quotations Marks around CC and CXX

### ISSUE THREAD

**Issue by KalleDK** [conan-io/conan#1169]
When I try to use the virtualenv on Windows in a cmd prompt the activate.bat has quotations marks around CC and CXX's values. This makes CMake sad apparently

```
CC="C:/Users/live/.conan/data/mingw_installer/0.1/lasote/testing/package/e3fafa852fbf3267e0e3fc81c4f41a7360e26aef/bin/gcc.exe"
ComSpec=C:\WINDOWS\system32\cmd.exe
CXX="C:/Users/live/.conan/data/mingw_installer/0.1/lasote/testing/package/e3fafa852fbf3267e0e3fc81c4f41a7360e26aef/bin/g++.exe"
```

```
CMake Error at C:/Program Files/CMake/share/cmake-3.7/Modules/CMakeDetermineCCompiler.cmake:48 (message):
  Could not find compiler set in environment variable CC:
 
"C:/Users/live/.conan/data/mingw_installer/0.1/lasote/testing/package/e3fafa852fbf3267e0e3fc81c4f41a7360e26aef/bin/gcc.exe".
Call Stack (most recent call first):
  CMakeLists.txt:1 (PROJECT)

CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
```

But if I remove the quotations marks in the bat file everything works

**Comment by KalleDK** [comment]
The workaround I use at the moment is this hack in the build.py where i give custom_runner to the ConanMultiPackager

```
def custom_runner(data):
    if data.startswith('activate.bat'):
        with fileinput.FileInput('activate.bat', inplace=True, backup='.bak') as file:
            for line in file:
                print(re.sub('(CC|CXX)="(.*)"', '\1=\2', line), end='')
    
    return os.system(data)
```

**Comment by memsharded** [comment]
We have to have a look at it. The problem with removing the quotes is when there are spaces in the path, which unfortunately happens more than desired. 
I also thought that the mingw_installer worked by putting the compiler in the PATH, not by setting CC and CXX, probably @lasote can give some feedback on it.

I have been able to reproduce doing:
```bash
$ conan install mingw_installer/0.1@lasote/testing -g virtualenv
$ activate
$ cmake -G "MinGW Makefiles"
```
It probably went unnoticed because using VS it doesn't happen. Labeling as bug, will try to make it into the release, but cannot guarantee. Thanks very much for reporting!

**Comment by lasote** [comment]
About the CC and CXX, you have to put the compiler in path but also to indicate the compiler executable name, so they are typically both needed.

**Comment by lasote** [comment]
We have released 0.21.2 fixing it. Please re-open if you detect any problem.
